azure-pipelines.yml

```diff
@@ -196,6 +196,8 @@ stages:
             sonar.analysis.pipeline=$(Build.BuildId)
             sonar.analysis.sha1=$(Build.SourceVersion)
             sonar.analysis.repository=$(Build.Repository.ID)
+            # needed for SonarCloud main branch decoration for unbound projects - see SC-3469
+            sonar.pullrequest.github.repository=$(Build.Repository.Name)
```

Comment:
Will the `$(Build.Repository.Name)` output sth like: `SonarSource/sonar-dotnet` ?

Reply:
I expect so, this is the environment variable which is used in scanner integration with AZP the https://github.com/SonarSource/sonar-scanner-vsts/blob/master/common/ts/prepare-task.ts#L12